src/Symfony/Component/PropertyAccess/Annotation/Adder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor adder configuration annotation.
+ *
+ * @Annotation
+ * @Target({"METHOD"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class Adder
```

Comment:
Hmmm... I've doing some experiments. If you omit `property` in the parameter still works because by default it assigns its value to the first public field in the annotation class. So, no code changes are needed. If we add another parameter, we just have to make sure it is not the first one in the class.